operations/mimir-mixin/dashboards/tenants.libsonnet

```diff
@@ -733,5 +733,45 @@ local filename = 'mimir-tenants.json';
           ],
         )
       )
+    )
+
+    .addRow(
+      ($.row('Compactions') + { collapse: true })
```

Comment:
What do you think of adding a panel showing the tenant's block count to this row as well?

Reply:
> ... we can move estimated compaction jobs + blocks count into separate row.

I was clearly confused here :)

Reply:
Added panel and [updated screenshot](https://private-user-images.githubusercontent.com/895919/307736258-59b1cf7a-e808-4aab-99ba-64bb178330bc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5NDA1NDgsIm5iZiI6MTcwODk0MDI0OCwicGF0aCI6Ii84OTU5MTkvMzA3NzM2MjU4LTU5YjFjZjdhLWU4MDgtNGFhYi05OWJhLTY0YmIxNzgzMzBiYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyNlQwOTM3MjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYmQwNjkyYzMwYjBiMTAwYjc5MjIxYmNkOTlhN2I2OTU3NjFmZDhiY2M2NmI4ZDEzNjM5Y2VkNzdiNDg4ZWE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.heubN2wq5MeXsDPMEjF2MwH3G2iTEPbnYbhQ6mlwKKE).